src/ui/public/courier/saved_object/saved_object_loader.js

```diff
@@ -103,7 +103,7 @@ export class SavedObjectLoader {
     return this.savedObjectsClient.find(
       {
         type: this.lowercaseType,
-        search,
+        search: search + '*',
```

Comment:
There's a reason we don't do this actually.  I'm not sure why it doesn't work but changing it ti have the * in front causes nothing to be returned for the dashboard list.

Until there is a better understanding of why the prepended * doesn't work (I'm pretty sure I started down this path before) I think we should stick to making this PR about fixing the regression.